iso/README.md

```diff
@@ -14,3 +14,9 @@ VBoxManage modifyvm foo --memory 1024 --vrde on --vrdeaddress 127.0.0.1 --vrdepo
 ```
 
 Then use the VirtualBox gui to start and open a session.
+
+## Release instructions
+ * Build an iso following the above build instructions.
+ * Test the iso with --iso-url=file:///$PATHTOISO.
+ * Push the new iso to GCS, with a new name (minikube-0x.iso).
```

Comment:
Yeah, it's just me with write access for now. I can add you if you'd like. I added instructions on how to push the iso as well.